examples/increment-only-counter/README.md

```diff
@@ -0,0 +1,36 @@
+# Increment-only counter
+
+This example implements an in-memory key-value cache.
+Keys are strings, and values are increment-only counters.
```

Comment:
The implementation here isn't really increment-only counters by the usual definition (e.g. https://github.com/pfraze/crdt_notes#state-based-increment-only-counter-g-counter), or what most people associate with the term. Instead what we have here is some sort of (Map of) Max Wins Register. Which doesn't strike me as particularly useful.